### ISSUE TITLE
Meaning of annotation fields.

### ISSUE THREAD

**Issue by 2006pmach** [oxuva/long-term-tracking-benchmark#7]
Hi, what are the exact meanings fo `contains_cuts`, `always_visible` and `object_presence`? Are the first two always the same per sequence while the last one might change from frame to frame? Thanks in advance.

**Comment by jvlmdr** [comment]
Yes. The `contains_cuts` and `always_visible` fields are per-video and per-track attributes, respectively. They specify whether or not the video contains cuts and whether or not the object is always visible. These are simply used for analysis purposes.

The `object_presence` is per-object per-frame and specifies whether the object is present. This must be predicted by the model.
